client/layout/guided-tours/tours/jetpack-plugin-updates-tour.js

```diff
@@ -0,0 +1,91 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import { query } from 'layout/guided-tours/positioning';
+
+export const JetpackPluginUpdatesTour = makeTour(
+	<Tour name="jetpackPluginUpdates" version="20180611">
+		{ /* Phantom step, wait for placeholder to disappear, then advance */ }
+		<Step
+			name="init"
+			target=".plugin-item.is-placeholder"
+			onTargetDisappear={ ( { next } ) => next() }
+			next="onLoaded"
+		/>
+		<Step
+			name="onLoaded"
+			wait={ () => !! query( '.plugin-item-jetpack .form-toggle:enabled' ).length }
+			target=".plugin-item-jetpack .form-toggle__switch"
+			onTargetDisappear={ /** Errors if missing */ () => {} }
```

Comment:
Magic, exhibit 2.

I think GT are buggy, while waiting it seems that the target appears and disappears. Ideally none of the logic would occur until wait passes.

Without `onTargetDisappear`, this errors as the component appears.

Reply:
Might want to add a code comment above this whole step, too.

/cc @lsinger for GT bugginess